### PR TITLE
Add my position

### PR DESCRIPTION
**Pull request by sonwng**
he he

### CODE REVIEW

**Comment by sonwng** [comment]
close #4758

**Comment by sonwng** [comment]
*so bad*

**Comment by sonwng** [comment]
# This is an <h1> tag
## This is an <h2> tag
###### This is an <h6> tag
This text will be italic*
_This will also be italic_

**This text will be bold**
__This will also be bold__

_You **can** combine them_
* Item 1
* Item 2
  * Item 2a
  * Item 2b
As Kanye West said:

> We're living the future so
> the present is our past.

I think you should use an
`<addr>` element here instead.
🏩

File: _pins/sonvidai.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: sonvidai
+latitude: 3333.333
+longtitude: 333.333
```
Comment (hectorsector) [inline _pins/sonvidai.json]: Great start, @sonwng! The CI tests are catching on to the fact that `longitude` is misspelled. You may also want to verify your coordinates.

**Comment by hollenberry** [comment]
Hey @sonwng ! Like Hector said, you might want to adjust your coordinates. After that, a fix of the spelling mistakes will be the final thing you need!

Please let us know if you need any help! We'll be closing this out to keep the repository clean if there's no action or conversation within the week.

**Comment by crichID** [comment]
@sonwng thank you for checking out our [on-demand classes](https://services.github.com/on-demand/). Since this file does not contain valid coordinates, I am going to go ahead and close this Pull Request. If you would like to complete the activity, please feel free to re-open this Pull Request or start again. If you have any questions, please let us know!

**Pull request closed without merging**